mailpoet/assets/js/src/automation/editor/index.tsx

```diff
@@ -42,7 +75,7 @@ function Editor(): JSX.Element {
     showIconLabels,
     workflow,
   } = useSelect(
-    (select) => ({
+    () => ({
```

Comment:
It feels strange to remove the `select` from the `useSelect` callback and instead rely on the imported `select`, but I'm not familiar enough with redux/wordpress data to know if there's a functional difference or if it could cause issues, so I'll ping @JanJakes for a more informed opinion.

Reply:
For context: `select` got imported here
https://github.com/mailpoet/mailpoet/pull/4462/commits/5b90216432a2f60802a3187b72efe67df3d1f95b#diff-644bd4c7e44dd87a74660f1465dcf0bfc1b11437d02a7b52ce4e001de1eccb96R6

I use it in the new `updateingActiveWorkflowNotPossible()` method. Codesniffer yelled, so I just removed it. Another way would be to use another variable name I guess.

My thinking in the end: The code in this PR will be removed quite soon :crossed_fingers:

Reply:
@websupporter Why did you remove it, in fact? You can just rename it, can't you? Either the import or the local variable. I'm not familiar with the internals of `useSelect` so I don't know what happens when you do this, but I've never seen it used like that in Gutenberg, so I think it would be safer to use the select passed to the hook, as @johnolek points out.

Reply:
Thank you. Updated in [dd8a252](https://github.com/mailpoet/mailpoet/pull/4462/commits/dd8a2525ae105aa3ed10be3c74abcecbda0980a2)